### ISSUE TITLE
Issue in Configmap of Free5gc components

### ISSUE THREAD

**Issue by lgabhishek18** [Orange-OpenSource/towards5gs-helm#93]
Hi folks, 

I have created a Kubernetes cluster (with 2 nodes) using kubeadm to deploy free5gc and ueransim.

Detailed info:
**Ubuntu:**  20.04.6 LTS
**Kernel version:** 5.4.0-152-generic
**Container Runtime version:** containerd://1.6.12
**K8s Version:** v1.20.1
**Free5gc:** v3.3.0
**Ueransim:** v3.2.6

When I install free5gc helm chart using: `helm -n free5gc install free5gc-v1 towards5gs/free5gc` my all pods were in CrashLoopBackOff state.

After, going through the various configurations I have found that there is issue a in configmap:

**Issue:**
```
logger:
  | -
    logger:  # log output setting
      enable: true  # true or false
      level: info  # how detailed to output, value: trace, debug, info, warn, error, fatal, panic
      reportCaller: false  # enable the caller report or not, value: true or false
```

**Solution:**
```
logger:  # log output setting
  enable: true  # true or false
  level: debug  # how detailed to output, value: trace, debug, info, warn, error, fatal, panic
  reportCaller: false  # enable the caller report or not, value: true or false
```
This is true for all compononts of free5gc i.e. amf, ausf, nrf, nssf, pcf, smf, udm, udr and webui.

Now, my all pods are in running state:

![image](https://github.com/Orange-OpenSource/towards5gs-helm/assets/60277134/fef24231-1f3a-4e7f-aa41-d0bae55be297)

I will raise the MR for the same.

Thanks!!

**Comment by lgabhishek18** [comment]
Issue in configmap is resolved

**Comment by lgabhishek18** [comment]
g

**Comment by raoufkh** [comment]
This has been fixed in 9946b896889ceecf4d4717a96c54f3aa04b212c1
